pandas/tests/indexing/test_at.py

```diff
@@ -234,3 +237,9 @@ def test_at_categorical_integers(self):
             for key in [0, 1]:
                 with pytest.raises(KeyError, match=str(key)):
                     df.at[key, key]
+
+    def test_at_does_not_expand(self):
```

Comment:
Maybe we should have two tests here, one for expansion in column axis and another one for expansion in index axis.